app/assets/stylesheets/modules/home.scss

```diff
@@ -0,0 +1,47 @@
+$good_dog_font: "GoodDog";
+
+.body-home-page {
+  background: url("dash.jpg") no-repeat center center fixed;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+}
+
+.new-bar {
+  padding: 0;
+}
+
+.bar {
+  max-width: 600px;
+}
+
+.search-button {
+  padding-left: 0;
+}
+
+.home-title {
+  -webkit-animation-duration: 1.7s;
```

Comment:
Properties should be ordered -webkit-animation-delay, -webkit-animation-duration, -webkit-animation-iteration-count, left, position, top<br>Avoid vendor prefixes.